local_costmap:
  global_frame: odom
  robot_base_frame: base_link
  update_frequency: 5.0

  # TODO what's the meaning of these parameters when the local costmap should never change (ie there are no dynamic obstacles)
  publish_frequency: 2.0
  static_map: false
  rolling_window: true
  width: 6.0
  height: 6.0
  resolution: 0.05
